redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -222,4 +222,81 @@ public static function logFile(): string
     {
         return rex_path::log('mail.log');
     }
+
+    public static function ErrorMail()
```

Comment:
```suggestion
    /**
     * @internal
     */
    public static function errorMail(): void
```

Die Klasse ist offizielle Api, die Methode soll man aber nicht selbst aufrufen. Daher würde ich sie als internal markieren.
Außerdem würde ich die Methode klein beginnen. Muss beim Aufruf dann auch noch angepasst werden.